pygenometracks/tracks/HiCMatrixTrack.py

```diff
@@ -215,12 +216,10 @@ def set_properties_defaults(self):
                                                     shape=self.hic_ma.matrix.shape)
             self.hic_ma.matrix = self.hic_ma.matrix + main_diagonal
 
-        self.norm = None
-
         self.process_color('colormap', colormap_possible=True,
                            colormap_only=True, default_value_is_colormap=True)
 
-        self.cmap = cm.get_cmap(self.properties['colormap'])
+        self.cmap = copy.copy(cm.get_cmap(self.properties['colormap']))
```

Comment:
Please double check if you want to have shallow copy or a deep one: https://docs.python.org/3/library/copy.html

Reply:
I followed the suggestion from matplotlib warning: 
```
MatplotlibDeprecationWarning: You are modifying the state of a globally registered colormap. In future versions, you will not be able to modify a registered colormap in-place. To remove this warning, you can make a copy of the colormap first. cmap = copy.copy(mpl.cm.get_cmap("bone_r"))
    self.cmap.set_bad('black')
```